src/vm/appdomain.cpp

```diff
@@ -8151,7 +8150,7 @@ void AppDomain::Exit(BOOL fRunFinalizers, BOOL fAsyncExit)
     LOG((LF_APPDOMAIN | LF_CORDB, LL_INFO10, "AppDomain::Domain [%d] %#08x %ls is exited.\n",
          GetId().m_dwId, this, GetFriendlyNameForLogging()));
 
-    ReJitManager::OnAppDomainExit(this);
+    CodeVersionManager::OnAppDomainExit(this);
```

Comment:
Switching the order does seem conceptually correct - I'll do that. In practice right now it won't matter because none of the code versions are being cleaned up. Historically for ReJit on desktop this looks like it should have been a legitimate issue. I'm guessing the overlap between ETW collection and ReJIT usage was small enough that nobody noticed but it isn't clear.

---
In reply to: [127332812](https://github.com/dotnet/coreclr/pull/12193#discussion_r127332812) [](ancestors = 127332812)